src/components/Popover.vue

```diff
@@ -29,7 +29,19 @@ interface ElementPosition {
 @description
 Implement a popover component
 
-@params {Boolean} [active = false] toggles visibility
+Use the popover inside a "safe" parent.
+To toggle the visibility use a `v-if` or a `v-show` on it's parent of any of it's ancestor.
+Warning: do not use the v-if or v-show directly on the popover component
+
+For example:
+```
+<span class="popover-container">
+  <popover>
+    <div> This will be displayed insidthe popover<div>
+  </popover>
+</span>
```

Comment:
This example does not showcase any v-if or v-show ; should we?

Reply:
I think so ! Thanks :)